x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -466,6 +488,38 @@ const AlertsTable: React.FunctionComponent<AlertsTableProps> = (props: AlertsTab
     ]
   );
 
+  const renderCellPopover = useMemo(
+    () =>
+      props.alertsTableConfiguration?.getRenderCellPopover
+        ? props.alertsTableConfiguration?.getRenderCellPopover({
+            context: renderCellContext,
+          })
+        : props.renderCellPopover,
```

Comment:
nitpick: think you can just use optional chaining all the way `getRenderCellPopover?.({})`